### ISSUE TITLE
Localization update

### ISSUE THREAD

**Issue by MichaelGrafnetter** [MichaelGrafnetter/yubikey-minidriver-admx#4]
I have added many new settings to the ADMX template. The French and Italian ADML files will need to be updated accordingly.

@serhiyresera @sha256-mercury If you have time, could you please translate the new strings?

**Comment by serhiyresera** [comment]
Yeah sure, I'll have a look into it next weekSent from my iPhoneOn 12 Jun 2024, at 23:58, Michael Grafnetter ***@***.***> wrote:﻿
I have added many new settings to the ADMX template. The French and Italian ADML files will need to be updated accordingly.
@serhiyresera @sha256-mercury If you have time, could you please translate the new strings?
